py3status/modules/wifi.py

```diff
@@ -66,16 +68,32 @@ class Py3status:
     cache_timeout = 10
     device = 'wlan0'
     down_color = 'bad'
-    format_down = 'W: down'
-    format_up = 'W: {bitrate} {signal_percent} {ssid}'
+    format = DEFAULT_FORMAT
     round_bitrate = True
     signal_bad = 29
     signal_degraded = 49
     use_sudo = False
 
     def post_config_hook(self):
+        # DEPRECATION WARNING
+        if not self.py3.get_placeholders_list(self.format):
```

Comment:
How about now? I can't test wifi stuffs atm. Laptop throwing a fit about `sudo` error code or something. Tried `use_sudo` and visudo fix. Nothing works right now. Meh.